### PR TITLE
MLPAB-377: Add fixtures page

### PR DESCRIPTION
**Pull request by acsauk**
# Purpose

Adds a fixtures page to allow setting donor flow state by section and links to jump into the CP flow either before or after the donor has IDed. This is aimed at helping non-devs/people-who-don't-like-query-strings to jump around the app when manual testing or quickly checking content.

Fixes MLPAB-377

### CODE REVIEW

File: app/internal/page/fixtures.go [hunk 1]
```diff
@@ -67,3 +105,201 @@ func MakeCertificateProvider(firstNames string) actor.CertificateProvider {
...
+	lpa.Tasks.YourDetails = TaskCompleted
+}
+
+func AddAttorneys(lpa *Lpa, count int) []string {
+	if count > len(AttorneyNames) {
+		count = len(AttorneyNames)
+	}
+
+	var firstNames []string
+	for i := 0; i < count; i++ {
```
Comment (hawx) [inline app/internal/page/fixtures.go:122]: if you change those maps to slices these could be neat `for name := range AttorneyNames[:count] { ... }`

File: app/internal/page/fixtures.go [hunk 2]
```diff
@@ -67,3 +105,201 @@ func MakeCertificateProvider(firstNames string) actor.CertificateProvider {
 		RelationshipLength:      "gte-2-years",
 	}
 }
+
```
Comment (hawx) [inline app/internal/page/fixtures.go:108]: I can't comment on the lines, but I've been getting paranoid about emails and mobile numbers, should we set them all to https://docs.notifications.service.gov.uk/rest-api.html#smoke-testing by default so people really can't mess up?

**Review by hawx — COMMENTED** [review]
Having a UI for this will make it so much easier to explain

File: app/internal/page/fixtures.go (hunk 2 above)
Comment (acsauk) [reply app/internal/page/fixtures.go:108]: Probably the safest bet yeah - I'll add a couple of consts in somewhere that we can reference where we need it in tests.

File: app/internal/page/fixtures.go [hunk 3]
```diff
@@ -1,11 +1,49 @@
...
+	"golang.org/x/exp/slices"
+
+	"github.com/ministryofjustice/opg-go-common/template"
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/actor"
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/date"
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/place"
 )
 
+var AttorneyNames = map[int]string{
```
[the remark this answers is not in the document]
Comment (acsauk) [reply app/internal/page/fixtures.go:23]: I was paranoid about the random order of slices when accessing them as the I'm relying on 0 and 1 being the names defined there to assert on - or am I getting this mixed up with another data structure that randomises things when accessed?

[the remark this answers is not in the document]
Comment (hawx) [reply app/internal/page/fixtures.go:23]: the only thing that randomises is the order of ranging over a map, accessing slices is fine they are just arrays (kinda)

**Review by hawx — APPROVED** [review]

**Review by hawx — APPROVED** [review]

**Pull request merged**